ServiceSection/index.html

```diff
@@ -19,7 +19,7 @@
   <nav class="navBar1">
     <div class="fullNav">
       <div id="logo">
-        <img src="../img/logo/navLogo.png" alt="AISKCON-Construction" />
+        <a href="/index.html"><img src="../img/logo/navLogo.png" alt="AISKCON-Construction" /></a>
```

Comment:
href="#" is used to link to the top of the current page! but we have to link the pages to the home Section so it can't be use (Can be used in the case of home section) .